zstash/create.py

```diff
@@ -102,7 +105,7 @@ def setup_create() -> Tuple[str, argparse.Namespace]:
     required.add_argument(
         "--hpss",
         type=str,
-        help='path to storage on HPSS. Set to "none" for local archiving. Must be set to "none" if the machine does not have HPSS access.',
+        help='path to storage on HPSS. Set to "none" for local archiving. Must be set to "none" if the machine does not have HPSS access. It also can be a Globus URL, e.g. globus://<UUID>/<PATH>. Names "alcf" and "nersc" are recognized as referring to the ALCF HPSS and NERSC HPSS endpoints.',
```

Comment:
Specify that "nersc"/"alcf" are valid for `<UUID>` but you still need the  `<PATH>`.